default_theme/index.js

```diff
@@ -71,21 +71,20 @@ module.exports = function (comments, options, callback) {
       }
     }
   };
+  
+  imports.renderSectionList =  _.template(fs.readFileSync(path.join(__dirname, 'section_list._'), 'utf8'), {
```

Comment:
resolved that.  I still get a unit test error.  Out of naivety, do you think it is related?